src/Entity/BasePostRepository.php

```diff
@@ -41,10 +42,15 @@ public function findLastPostQueryBuilder($limit)
      */
     public function countCommentsQuery($post)
     {
-        return $this->getEntityManager()->createQuery('SELECT COUNT(c.id)
-                                          FROM Application\Sonata\NewsBundle\Entity\Comment c
-                                          WHERE c.status = 1
-                                          AND c.post = :post')
-                    ->setParameters(['post' => $post]);
+        $qb = $this->createQueryBuilder('c');
+
+        return $qb
+            ->select('COUNT(c.id)')
+            ->where($qb->expr()->eq('c.status', ':status'))
```

Comment:
isnt it clearer to just use `c.status = :status`? This comment is not blocking, just a question

Reply:
Both syntaxes are clear enough to me, since I'm used to work with the query builder.
I've chose to keep the query builder syntax for the comparison too in order to avoid a mix with DQL.